source/particle/output/ascii.cc

```diff
@@ -106,19 +99,15 @@ namespace aspect
             output << "\n";
           }
 
-        file_index++;
+        this->increment_file_index ();
 
-        return output_file_prefix;
+        return "txt";
       }
 
       template <int dim>
       template <class Archive>
-      void ASCIIOutput<dim>::serialize (Archive &ar, const unsigned int)
-      {
-        // invoke serialization of the base class
-        ar &file_index
-        ;
-      }
+      void ASCIIOutput<dim>::serialize (Archive &, const unsigned int)
+      {}
```

Comment:
I'm going to note that this is now actively incorrect because you don't serialize the member variables of the base class.